docs/data_dictionaries/index.rst

```diff
@@ -3,20 +3,38 @@
 Data Dictionaries
 =================
 
+Data Processed & Cleaned by PUDL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The PUDL data dictionary provides detailed information about the output tables,
+including field names, data types, and descriptions
+for each field within the respective tables.
+
 .. toctree::
-   :caption: Data Processed & Cleaned by PUDL
    :maxdepth: 1
+   :titlesonly:
 
    pudl_db
 
+Raw, Unprocessed Data
+^^^^^^^^^^^^^^^^^^^^^
+This section outlines the raw, unprocessed data from the FERC forms,
```

Comment:
```suggestion
For data sources where substantial effort is required to publish raw databases (e.g., FERC Form 1), the dictionaries below provide key metadata on the raw data sources (e.g., table names and descriptions).
```
This section could grow to include metadata on other raw sources we're publishing, so maybe it's worth keeping it more general.